src/app/app.filter.pipe.ts

```diff
@@ -1,11 +1,15 @@
 import { Pipe, PipeTransform } from '@angular/core';
+import { escapeRegexp, regexpFromArray } from 'ng2-qgrid/core/utility/kit';
```

Comment:
u can't do it, only import from `ng2-qgrid` is allowed

just add this function on application level not lib.